### PR TITLE
Logs nvidia-gpu device plugin under /var/log/.

### PR DESCRIPTION
**Pull request by jiayingz**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by roberthbailey** [comment]
please add a release note stanza to the PR description

File: cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml [hunk 1]
```diff
@@ -35,9 +35,15 @@ spec:
       - name: dev
         hostPath:
           path: /dev
+      - name: varlog
+        hostPath:
+          path: /var/log
       containers:
       - image: "gcr.io/google-containers/nvidia-gpu-device-plugin@sha256:0e79da6998a61257585e0d3fb5848240129f0fa5b4ad972dfed4049448093c33"
-        command: ["/usr/bin/nvidia-gpu-device-plugin", "-logtostderr"]
```
Comment (mindprince) [inline cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml]: Would I still be able to use `kubectl logs` to see the logs?

Also, why is this needed at all? Doesn't logging to stderr automatically go to stackdriver etc.?

**Comment by mindprince** [comment]
This is not needed.

/close
